### PR TITLE
[deckhouse] Added validation of moduleconfigs parameter update.windows

### PR DESCRIPTION
**Pull request by Suselz**
## Description
Added validation of the moduleconfigs settings about update windows for the `deckhouse` module. 
Also brought the code of the modified file to a single style (parameterization of the object name was added)

## Why do we need it, and what problem does it solve?
This solves the problems described in issue [#9080](https://github.com/deckhouse/deckhouse/issues/9080)

## What is the expected result?
of the update.windows.from field is greater than the value of the update.windows.to field, so that when incorrect parameters are specified, the values ​​of the moduleconfig object are not written

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [ ] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries

```changes
section: deckhouse
type: feature
summary: Added validation of `update.windows` module parameter.
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Suselz. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Suselz. I've started the workflow for label 'e2e/run/yandex-cloud'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @Suselz. I've started the workflow for label 'e2e/run/yandex-cloud'...

File: modules/002-deckhouse/templates/validation.yaml [hunk 1]
```diff
@@ -100,3 +101,45 @@ spec:
...
+      operations:  ["CREATE", "UPDATE"]
+      resources:   ["moduleconfigs"]    
+  validations:
+    - expression: '!(object.metadata.name == "deckhouse" && has(object.spec.settings.update) && has(object.spec.settings.update.windows) && object.spec.settings.update.windows.all(w, int(string(w.from).replace(":", "")) > int(string(w.to).replace(":", ""))))'
+      reason: Forbidden
+{{- if semverCompare ">= 1.27" .Values.global.discovery.kubernetesVersion }}
+      messageExpression: '''Invalid update windows. Start time (windows.to) should be less than the end time of the update window (windows.from)'''         
+{{- end }}
+---
+{{- if semverCompare ">= 1.28" .Values.global.discovery.kubernetesVersion }}
```
Comment (yalosev) [inline modules/002-deckhouse/templates/validation.yaml:132]: It has to be `admissionregistration.k8s.io/v1` for 1.30+ kubernetes

**Review by yalosev — APPROVED** [review]

File: modules/002-deckhouse/templates/validation.yaml (hunk 1 above)
Comment (Suselz) [reply modules/002-deckhouse/templates/validation.yaml]: Аdded comparison kubernetesVersion >= 1.30 for each object `ValidatingAdmissionPolicyBinding`

**Pull request merged**